### ISSUE TITLE
Mvn build fails with Java 8 

### ISSUE THREAD

**Issue by soisik** [lsc-project/lsc#4]
Environment : Apache Maven 3.3.9, JDK version: 1.8.0_111
Command : mvn -opendj clean package

Some minor adaptations of pom.xml are to be made so LSC can be built successfully with java 8 without having to switch to JDK 7 or 6.

### bc-jdk-version property missing,

Solution : simply adding a jdk18 profile based on jdk17 profile.

#### Problem
During maven-jaxb2-plugin generate-sources phase we have this error :
```
org.xml.sax.SAXParseException; systemId: jar:file:/home/sfroger/.m2/repository/com/sun/xml/bind/jaxb-xjc/2.2.6/jaxb-xjc-2.2.6.jar!/com/sun/tools/xjc/reader/xmlschema/bindinfo/binding.xsd; lineNumber: 52; columnNumber: 88; schema_reference: Failed to read schema document 'xjc.xsd', because 'file' access is not allowed due to restriction set by the accessExternalSchema property.
at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:203)
```
See https://java.net/jira/browse/MAVEN_JAXB2_PLUGIN-80. 

#### Solutions
- Quick fix : Use the workaround provided in bug report (2.1.4)
- Long term fix : Upgrade maven-jaxb2-plugin from 0.8.0 to 0.9.0+ : works but I'd like to further analyze for potential side effects, because generated lsc.episode and WS API are differents when generated with newer plugins; There will be a specific issue for libraries upgrade (2.2.0)

### Javadoc generation fails when HTLM do not validate.

See http://stackoverflow.com/questions/15886209/maven-is-not-working-in-java-8-when-javadoc-tags-are-incomplete

- Quick fix : These errors can be ignored by the plugin by adding a property to jdk18 profile et passing it to the plugin during java doc generation <javadoc.opts>-Xdoclint:none</javadoc.opts> (2.1.4)

- Long term fix : Correct the errors and warnings in the javadoc commentaries throughout source code.
There will be a specific issue for this task (2.2.0)

**Comment by soisik** [comment]
Fixed in 2.1.4

**Comment by soisik** [comment]
Fixed by merge in 2.2.0
